config.default.yaml

```diff
@@ -166,6 +185,10 @@ renewable:
     hydro_max_hours: "energy_capacity_totals_by_country" # one of energy_capacity_totals_by_country, estimate_by_large_installations or a float
     clip_min_inflow: 1.0
 
+conventional: 
+  nuclear:
+    energy_availability_factors: "data/nuclear-eafs.csv" # float of file name
```

Comment:
This should maybe show up in `snakemake.inputs` of the rule where it is used.

Option A: make this config setting a boolean, fixed file reference to `data/nuclear-eafs.csv` in Snakefile

Option B: refer to file referenced here as optional input in `snakemake.inputs`: `eaf=config["conventional"]["nuclear"]["energy_availability_factors"] if config["conventional"]["nuclear"]["energy_availability_factors"] else []

Reply:
Now with the general implementation, I wouldn’t insist on handling all the possible file references in Snakefile.